provisionerd/runner/runner.go

```diff
@@ -629,7 +629,7 @@ func (r *Runner) runTemplateImportParse(ctx context.Context) (
 type templateImportProvision struct {
 	Resources             []*sdkproto.Resource
 	Parameters            []*sdkproto.RichParameter
-	ExternalAuthProviders []string
+	ExternalAuthProviders []*sdkproto.ExternalAuthProviderResource
```

Comment:
when you convert this type to `proto.CompletedJob_TemplateImport` on line 570, you also need to set the ExternalAuthProviderNames for compatibility with back-level Coderd.